recipes/libsigcpp/2.10.x/test_package/conanfile.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
```

Comment:
```suggestion
```

Reply:
Why should I remove this? https://github.com/conan-io/conan-center-index/blob/master/recipes/libjpeg/all/conanfile.py#L1-L2
Here it is present